commands/systems/http_push.py

```diff
@@ -86,8 +86,6 @@ def display_results(self, redfishConfig):
             Trace.log(TraceLevel.INFO, '[] Status       : {}'.format(self.link.urlStatus))
             Trace.log(TraceLevel.INFO, '[] Reason       : {}'.format(self.link.urlReason))
 
-        if self.context != '':
-            print(' [] Context    : {}'.format(self.context))
```

Comment:
Please make a small change to use: Trace.log(TraceLevel.INFO, ' [] Context : {}'.format(self.context))

Reply:
Hi Joe, there is no variable named "context" in CommandHandler class, so isn't it worth removing it?

OR,
I can also do,
if self.context:
            Trace.log(TraceLevel.INFO, ' [] Context : {}'.format(self.context))

Reply:
Yes, please do your OR statement, where you are using Trace.log instead of print. Thanks.